pcsx2/GameDatabase.cpp

```diff
@@ -16,54 +16,93 @@
 #include "PrecompiledHeader.h"
 
 #include "GameDatabase.h"
+#include "Config.h"
+#include "Host.h"
+
+#include "common/FileSystem.h"
+#include "common/Path.h"
+#include "common/StringUtil.h"
+#include "common/Timer.h"
 
 #include "fmt/core.h"
 #include "fmt/ranges.h"
 #include "yaml-cpp/yaml.h"
 #include <fstream>
 #include <algorithm>
+#include <mutex>
+#include <thread>
+
+static constexpr char GAMEDB_YAML_FILE_NAME[] = "GameIndex.yaml";
+static constexpr char GAMEDB_CACHE_FILE_NAME[] = "gamedb.cache";
+static constexpr u64 CACHE_FILE_MAGIC = UINT64_C(0x47414D4544423031); // GAMEDB01
```

Comment:
Leftover from the dropped Gamedb cache?

The commit `GameDatabase: Move out of gui and add cache` should also not mention a cache if none is added

Reply:
Additionally, I believe `Config: Add cache folder` can be dropped

Reply:
No, the cache is used for shaders and the game list in later commits.

Reply:
Ah, I only did a quick look though and didn't see anything using the cache folders yet, my mistake